mapper/mapper.go

```diff
@@ -470,3 +472,101 @@ func MapCookiePreferences(req *http.Request, preferencesIsSet *bool, policy *mod
 		Usage:     preferencesCookie.Policy.Usage,
 	}
 }
+
+func CreateDatasetPage(ctx context.Context, req *http.Request, d zebedee.Dataset, dlp zebedee.DatasetLandingPage, bc []zebedee.Breadcrumb, versions []zebedee.Dataset, lang string, apiRouterVersion string) datasetPage.Page {
+	dp := datasetPage.Page{}
+
+	MapCookiePreferences(req, &dp.Page.CookiesPreferencesSet, &dp.Page.CookiesPolicy)
+	dp.Type = "dataset_page"
```

Comment:
Not sure, if I had to guess maybe because there are so many dataset types pages the standard was given the page suffix to try and reduce confusion when talking about dataset_timeseries or dataset_reference_tables for example? @mr-nick17 @AngelaLW any idea?

Reply:
I don't have any recollection of a discussion about the _page suffix, but I did ask Steve about it last week and he said we should be following the type set in the data.json as per the other pages, so the type should be 'dataset' in this instance.